.changeset/poor-years-tickle.md

```diff
@@ -0,0 +1,15 @@
+---
+'@commercetools-applications/merchant-center-custom-view-template-starter-typescript': patch
+'@commercetools-applications/merchant-center-template-starter-typescript': patch
+'@commercetools-frontend/application-components': patch
+'@commercetools-applications/merchant-center-custom-view-template-starter': patch
+'@commercetools-applications/merchant-center-template-starter': patch
+'@commercetools-website/components-playground': patch
+'@commercetools-frontend/react-notifications': patch
+'@commercetools-frontend/application-shell': patch
+'@commercetools-website/custom-views': patch
+'@commercetools-frontend/i18n': patch
+'@commercetools-website/custom-applications': patch
+---
+
+Update all ui-kit packages to v17.0.0
```

Comment:
Is this changeset correct? The update was already done, I think we can remove this changeset

Reply:
Yes, good point, thanks - [8ddb756](https://github.com/commercetools/merchant-center-application-kit/pull/3403/commits/8ddb7562c30e883ccb3c2d6eff05201c6c50e487)